force-app/main/service/DonationHistoryService.cls

```diff
@@ -90,10 +90,11 @@ public with sharing class DonationHistoryService {
     public List<Opportunity> getDonationHistory(Id contactId, Integer offset, String filter) {
         if (hasAccessToDonationHistory()) {
             List<Opportunity> credits = null;
+            String amount = UTIL_Currency.getInstance().isMultiCurrencyOrganization() ? 'convertCurrency(Amount)' : 'Amount' ;
```

Comment:
This is similar to the method that could get called from within the query.

Reply:
Done